frontend/javascripts/router.tsx

```diff
@@ -603,46 +612,30 @@ class ReactRouter extends React.Component<Props> {
                   return <FinishResetPasswordView resetToken={params.token} />;
                 }}
               />
-              <Route
-                path="/datasets/:organizationId/:datasetName/view"
-                render={this.tracingViewMode}
-              />
+              {/*legacy view mode route */}
               <RouteWithErrorBoundary
-                path="/datasets/:id/view"
-                render={({ match, location }: ContextRouter) => (
-                  <AsyncRedirect
-                    redirectTo={async () => {
-                      const datasetName = match.params.id || "";
-                      const organizationId = await getOrganizationForDataset(datasetName);
-                      return `/datasets/${organizationId}/${datasetName}/view${location.search}${location.hash}`;
-                    }}
-                  />
-                )}
+                path="/datasets/:organizationId/:datasetName/view"
+                render={this.tracingViewModeLegacy}
               />
+              <Route path="/datasets/:datasetNameAndId/view" render={this.tracingViewMode} />
+              {/*maybe this also needs a legacy route?*/}
```

Comment:
Yes I would think so, at least there are such links in the wild, for example in the [connectome viewer repository](https://github.com/scalableminds/connectome-viewer#:~:text=https%3A//webknossos.org/datasets/scalable_minds/l4dense_motta_et_al_demo_v2/sandbox/skeleton) and I've also sent such links via email to clients.